src/styles/main.scss

```diff
@@ -1,3 +1,23 @@
-body {
+@import "./utils/_variables.scss";
+@import "./utils/_functions.scss";
+@import "./blocks/card.scss";
+@import "./blocks/content.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/stars.scss";
+
+html {
+  font-family: $main-font;
+}
+
+body,
+p,
+h2 {
   margin: 0;
+  padding: 0;
+}
+
+main {
```

Comment:
Please ensure that the 'main' element is used in the HTML file according to the task requirements.